content/outcomes/cloud-native-development/general__spring_ecosystem.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Spring Ecosystem
+weight: 30
+layout: single
+team:
+ - VMware Tanzu Labs
+---
+
+We lean heavily on the [open-source ecosystem of Spring projects](https://spring.io/) to replace vendor- or container-specific technologies or proprietary solutions for Java-based applications. There are many specific tools and frameworks under that banner. We will show you the best ones to solve your specific problems during the engagement. The two to know beforehand are Spring Boot and Spring Cloud, as they each provide a common core that we use throughout the engagement.
+
+## Spring Boot
+Spring Boot provides a “convention over configuration” foundation for Spring-based applications. Follow [this guide](https://spring.io/guides/gs/spring-boot/) to build a simple RESTful web service using Spring Boot and minimal configuration.
+
+## Spring Cloud
+
+<<<<<<< HEAD
+[Spring Cloud](http://projects.spring.io/spring-cloud/) provides a set of abstractions and integrations that enable best practice cloud-native behavior, where we cover the actual implementation details for common tasks like externalizing configuration in a [Spring Cloud Config Server](https://cloud.spring.io/spring-cloud-config/reference/html/#_spring_cloud_config_server), using service discovery, and API gateways other route services.
+Refer to the product documentation to become familiar with the basic concepts from and to better understand tools you have in your tool belt once we get started.
+=======
+[Spring Cloud](http://projects.spring.io/spring-cloud/) provides a set of abstractions and integrations that enable best practice cloud native behavior. We will cover the actual implementation details for common tasks like externalizing configuration in a [Spring Cloud Config Server](https://cloud.spring.io/spring-cloud-config/reference/html/#_spring_cloud_config_server), using service discovery, and API gateways other route services.
+Here, you should get familiar with the basic concepts from their documentation to understand better what tools you have in your tool belt once we get started.
+>>>>>>> 244d986b9dee0e461cd063af7babd7b86e487dff
```

Comment:
Fix conflict.

Reply:
@lizz7000 @Maarc Do you happen to know what happened?

Reply:
@lizz7000 brought those. They have been fixed now in 961ee163a68737d1eb20b82c27aaa8537cdb402e